### DISCUSSION TITLE
WHY IT WONT INSTALL??????

### DISCUSSION THREAD

**Discussion by libracrumbs** [KadeArchive/Kade-Engine#2680] [Q&A] [answered]
I TRIED EVERYTHING IT JUST DOES NOT WORK PLS HELP I JUST WANT TO BUILD THE ENGINE IS IT TOO MUCH TO ASK FOR??? also can someone tell me where the git paths are? it thinks my drive is E:/ but it is D:/ 
![image](https://user-images.githubusercontent.com/45106924/140516230-3d1459ce-f1f7-42e8-9615-e9c90dec0ae3.png)

**Comment by hifishlol** [comment, marked as answer]
i dont know how to fix this but ill try my best

go to your d drive and open it in command prompt or powershell
then try to add the libraries/dependencies

**Reply by libracrumbs** [reply]
Thank you, but i actually found it before and forgot to remove it, i just manually deleted every single piece of registry code + uninstall tool
